src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Setter.cs

```diff
@@ -161,6 +161,7 @@ public DependencyProperty Property
         /// </summary>
         [System.Windows.Markup.DependsOn("Property")]
         [System.Windows.Markup.DependsOn("TargetName")]
+        [DesignerSerializationVisibility(DesignerSerializationVisibility.Content)]
```

Comment:
Here we are changing the `DesignerSerializationVisibility.Default` to `DesignerSerializationVisibility.Content` which will cause behavioral change for the code and needed to documented.

Reply:
Added in the documentation